src/calculateRentalCost.js

```diff
@@ -28,7 +28,27 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const CAR_DAY = 7;
+  const CAR_DAY_SIX = 6;
+  const CAR_DAY_THREE = 3;
+  const CAR_DAY_DISKONT = 230;
+  const CAR_DAY_DISKONT_SIX = 220;
+  const CAR_DAY_DISKONT_THREE = 100;
```

Comment:
don't hardcode these values, you need to count them
ex
`const basePrise = days * PRICE_PER_DAY;`

and then return
`return basePrise - LONG_TERM_DISCOUNT;`